style.css

```diff
@@ -1,6 +1,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
 
 @media (min-width: 320px) {
+  body {
+    margin: 0;
+    overflow-x: hidden;
+  }
+
```

Comment:
- [ ] When importing a font, please know that you can import several styles for a single font family for example here in google fonts, there is this style:

![image](https://user-images.githubusercontent.com/20567503/185681661-37994013-94db-45ce-8456-3bdb7ccb7153.png)

If you import this you'll be able to use the `font-weight: 500` as you meant to do, and this will fix your issue so kindly import all the styles you want to use so you can use the correct weight.